web/modules/custom/joinup_front_page/joinup_front_page.module

```diff
@@ -12,8 +12,21 @@ declare(strict_types = 1);
  */
 function joinup_front_page_preprocess_block(&$variables): void {
   if (\Drupal::currentUser()->isAuthenticated()) {
-    // The libraries are available for authenticated users.
-    return;
+    // The libraries are available for all users except the
+    // ones only with authenticated role.
```

Comment:
This documentation is not clear, it assumes the reader knows exactly what this is about. Let's update this so we have a good explanation why we need to alter a certain block.

Reply:
Here is a proposal for updated documentation:

```
/**
 * Implements hook_preprocess_HOOK().
 *
 * The 'Discover topics' block which is shown on the front page is created using
 * the Views module. The block includes a 'See more topics' link which opens a
 * modal dialog with the full list of available topics. Include the necessary JS
 * so that all users can interact with the modal.
 */
```